housekeep.py

```diff
@@ -144,26 +175,64 @@ def dispatch_superclean():
 
 
 def get_opts(argv):
+    # From: https://stackoverflow.com/a/49999185/149900
+    class NoAction(argparse.Action):
+        def __init__(self, **kwargs):
+            kwargs.setdefault("default", argparse.SUPPRESS)
+            kwargs.setdefault("nargs", 0)
+            super().__init__(**kwargs)
+
+        def __call__(self, *args, **kwargs):
+            pass
+
+    dispers = {
+        name.replace("dispatch_", ""): inspect.getdoc(obj)
+        for name, obj in inspect.getmembers(sys.modules[__name__])
+        if name.startswith("dispatch_") and inspect.isfunction(obj)
+    }
+
     parser = argparse.ArgumentParser()
+    parser.register("action", "no_action", NoAction)
+
     parser.add_argument(
-        "cmd", metavar="CMD", choices=["setup", "cleanup", "superclean"]
```

Comment:
New code now autodetects available `dispatch_*` funcs. SSOT (Single Source Of Truth) and all that :-)